src/pages/core/deserialization.md

```diff
@@ -0,0 +1,158 @@
+- **Update** Authority: The authority over the asset, or the collection Address if it's part of one 
+- **Name**: The Asset Name
+- **Uri**: The uri to the asset off-chain metadata.
+
+{% dialect-switcher title="Deserialize an Asset" %}
+
+{% dialect title="JavaScript" id="js" %}
+```ts
+const accountData = await umi.rpc.getAccount(
+  publicKey('11111111111111111111111111111111')
+)
+
+if (!accountData.exists) throw 'Account does not exist'
+
+const assetV1 = deserializeAssetV1(accountData)
+
+console.log({ assetData })
+```
+{% /dialect %}
+
+{% dialect title="Rust" id="rust" %}
+```rust
+let rpc_client = rpc_client::RpcClient::new("https://api.devnet.solana.com".to_string());
+
+let asset = Pubkey::from_str("11111111111111111111111111111111").unwrap();
+
+let account = rpc_client.get_account(&asset).await.unwrap();
+
+let asset_v1 = BaseAssetV1::from_bytes(&account.data).unwrap();
+
+println!("assetV1: {:?}", asset_v1);
+```
+{% /dialect %}
+
+{% dialect title="Anchor" id="anchor" %}
+```rust
+let asset = ctx.accounts.asset;
+
+let data = asset.try_borrow_data()?;
+
+let asset_v1 = BaseAssetV1::from_bytes(&data.as_ref())
+
+println!("assetV1: {:?}", asset_v1);
+```
+{% /dialect %}
+
+{% /dialect-switcher %}
+
+### Deserializing Collections
+
+Deserializing the `Collection` account will return information about:
+
+- **Update** Authority:	The authority over the collection and all the asset inside of it
+- **Name**:	The collection name.
+- **Uri**:	The uri to the collections off-chain metadata.
+- **Num Minted**: The number of assets minted in the collection.
+- **Current size**:	The number of assets currently in the collection.
+
+{% dialect-switcher title="Deserialize a Collection" %}
+{% dialect title="JavaScript" id="js" %}
+
+```ts
+const accountData = await umi.rpc.getAccount(
+  publicKey('11111111111111111111111111111111')
+)
+
+if (!accountData.exists) throw 'Account does not exist'
+
+const collectionV1 = deserializeCollectionV1(accountData)
+
+console.log({ assetData })
+```
+
+{% /dialect %}
+
+{% dialect title="Rust" id="rust" %}
+
+```rust
+let rpc_client = rpc_client::RpcClient::new("https://api.devnet.solana.com".to_string());
+
+let collection = Pubkey::from_str("11111111111111111111111111111111").unwrap();
+
+let account = rpc_client.get_account(&collection).await.unwrap();
+
+let collection_v1 = CollectionV1::from_bytes(&account.data).unwrap();
+
+println!("collection_V1: {:?}", asset_v1);
+```
+
+{% /dialect %}
+
+{% dialect title="Anchor" id="anchor" %}
+
+```rust
+let collection = ctx.accounts.collection;
+
+let data = collection.try_borrow_data()?;
+
+let collection_v1 = BaseCollectionV1::from_bytes(&data.as_ref())?;
+
+println!("collection_V1: {:?}", asset_v1);
+```
+
+{% /dialect %}
+
+{% /dialect-switcher %}
+
+### Deserializing Plugins
+
+Even if the plugin data is saved directly on the Asset account, because of the size of it we should deserialize and save just the singular plugin data
```

Comment:
I think this wording should be changed, because the plugin data is stored on the asset account, and it is included in the Asset type.  But I think this wording will probably change based on my initial comment anyways.